### PR TITLE
success of adding the reequest

### PR DESCRIPTION
**Pull request by goodbiruk**
ready to review

### CODE REVIEW

File: src/app/services/request.service.ts [hunk 1]
```diff
@@ -1,9 +1,9 @@
 import { Injectable } from '@angular/core';
-import { Observable } from 'rxjs';
```
Comment (practicalit) [inline src/app/services/request.service.ts:2]: What is this change about?

File: src/app/pages/request/request.component.ts [hunk 2]
```diff
@@ -59,11 +59,11 @@ export class RequestComponent implements OnInit {
     
     this.requestService.postRequest(request).subscribe(
       response => {
-        this.success = false;
-        if (response.success) {
+        if (response === null) {
```
Comment (practicalit) [inline src/app/pages/request/request.component.ts:62]: What is this?? If the response is null, we are going to accept it as success??

File: package.json [hunk 3]
```diff
@@ -18,11 +18,11 @@
     "@angular/core": "9.0.5",
     "@angular/forms": "9.0.5",
     "@angular/http": "7.2.16",
-    "@angular/localize": "^9.0.5",
```
Comment (practicalit) [inline package.json:21]: What is the need for these changes at all?

**Comment by practicalit** [comment]
This happens to be the backend issue and and is to be moved to backend developers.

**Comment by goodbiruk** [comment]
sorry,
i try to fix but, i don't have access backend

**Pull request closed without merging**